### DISCUSSION TITLE
mishards: [docker run or docker-compose] start up failed...

### DISCUSSION THREAD

**Discussion by tinygiants-tech** [milvus-io/milvus#7011] [Q&A and General discussion]
start up mishards through "docker run ...  docker-compose" failed

command details:
docker run -d --restart=always --name mishards -p 19531:19531 -p 19532:19532 -v /home/$USER/docker/mishards/db:/tmp/milvus/db -v /home/$USER/docker/mishards/conf/mishards_env:/source/mishards/.env mishards python3 mishards/main.py
or 
docker-compose -f cluster_mishards.yml up

terminal feedback:
docker: Error response from daemon: invalid header field value "oci runtime error: container_linux.go:247: starting container process caused "exec: \"python3\": executable file not found in $PATH"\n".
or
ERROR: for mishards  Cannot start service mishards: invalid header field value "oci runtime error: container_linux.go:247: starting container process caused \"exec: \\\"python3\\\": executable file not found in $PATH\"\n"
ERROR: Encountered errors while bringing up the project.

now track -> echo $PATH exists "/usr/local/python3/bin"
cd /usr/local/python3/bin

ll| grep python3

lrwxrwxrwx 1 root root 9 Aug 3 17:04 python3 -> python3.7
-rwxrwxrwx 2 root root 14000911 Aug 3 17:04 python3.7
lrwxrwxrwx 1 root root 17 Aug 3 17:04 python3.7-config -> python3.7m-config
-rwxrwxrwx 2 root root 14000911 Aug 3 17:04 python3.7m
-rwxrwxrwx 1 root root 2910 Aug 3 17:04 python3.7m-config
lrwxrwxrwx 1 root root 16 Aug 3 17:04 python3-config -> python3.7-config

Looking forward to reply!

**Comment by yhmo** [comment]
"oci runtime error: container_linux.go", the error looks like the docker is not compatible with the linux core.
